index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title></title>
+             <meta charset="utf-8" />
+        <link href="css/styles.css" type="text/css"
+              rel="stylesheet" />
+    </head>
+
+<body>
+
+<header>
+    <h1>H&amp;M</h1>
+    <a href="index.html">
+      <img src=images/logo.png />
+    </a>
+ <nav>
+        <ul>
+            <li><a href="#">Ladies</a></li>
+            <li><a href="#">Men</a></li>
+            <li><a href="#">Kids</a></li>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Sale</a></li>
+        </ul>
+    </nav>
+</header>
+
+<div class="row products">
+
+    <h2>Cardigans</h2>
+
+        <ul>
+
+            <li>
+                <img class="product-img" src="images/img1.png" />
```

Comment:
As with the original submission of this project, nice microformatting here on these items!